packages/components/timepicker/src/components/osds-timepicker/osds-timepicker.tsx

```diff
@@ -10,11 +13,16 @@ import { DEFAULT_ATTRIBUTE } from './constants/default-attributes';
 })
 export class OsdsTimepicker implements OdsTimepickerAttribute {
   @Element() el!: HTMLElement;
-  inputEl?: HTMLInputElement;
+  input: (OsdsInput & HTMLElement) | HTMLInputElement | null = null;
```

Comment:
I think the type is just OsdsInput | null

Reply:
HTMLInputElement is still necessary to access to value